v1/tarball/image_test.go

```diff
@@ -58,7 +58,7 @@ func TestBundleSingle(t *testing.T) {
 		t.Fatalf("Error loading image: %v", err)
 	}
 	if _, err := img.Manifest(); err == nil {
-		t.Fatalf("Error expected loading manifest.")
+		t.Fatalf("Error expected loading manifest: %v", err)
```

Comment:
LOL, I meant to print the manifest.  This will always print `nil` :)

Reply:
D'oh. Shouldn't code before coffee...